FerroSoftware.FtpUse version 2.2

Masamune3210 opened the pull request:
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101858)

Masamune3210 committed:
FerroSoftware.FtpUse version 2.2

Trenly commented on manifests/f/FerroSoftware/FtpUse/2.2/FerroSoftware.FtpUse.locale.en-US.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageLocale: en-US
+Publisher: Ferro Software
+PackageName: FtpUse
+License: Copyright © 2000-2022 FERRO Software
```
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

Trenly commented on manifests/f/FerroSoftware/FtpUse/2.2/FerroSoftware.FtpUse.installer.yaml:
```diff
@@ -0,0 +1,12 @@
+- InstallerUrl: https://www.ferrobackup.com/download/FtpUseInst.exe
+  Architecture: x86
+  InstallerType: inno
+  InstallerSha256: E087AF89FE74EB062AA75A7D189E20E2B2BE80BC815356B987E06F5DE98A15C9
```
```suggestion
  InstallerSha256: E087AF89FE74EB062AA75A7D189E20E2B2BE80BC815356B987E06F5DE98A15C9
  ProductCode: '{7061EF03-4B2C-4707-B7FD-20059AE5F6EB}_is1'
```

Masamune3210 committed:
Fix license

No license is declared on the site as far as I can see, defaulting to Proprietary

Masamune3210 committed:
Update manifests/f/FerroSoftware/FtpUse/2.2/FerroSoftware.FtpUse.installer.yaml

Co-authored-by: Kaleb Luedtke <trenlymc@gmail.com>

stephengillie commented:
This package is expecting arguments:
```

FtpUse v2.2 - Mounts remote storage as a local hard disk by FTP protocol
Copyright (C) 2015 Ferro Software - www.ferrobackup.com

The syntax of this command is:

FTPUSE devicename [ftphostname[/remotepath] [password |]]
        [/USER:username]
        [/NOPASSIVE]
        [/OWNERACCESSONLY]
        [/PORT:portnumber]
...
```
